blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ None
 Changes
 =======
 
+- Removed setting: ``cluster.graceful_stop.reallocate`` as it had no effect
```

Comment:
I'm wondering if we should instead mark it as deprecated and only remove it in `4.0`. Otherwise CrateDB won't start up in 3.3 if people have it in their configuration file and forget to remove it.

Reply:
Yeah, I guess that makes sense. Didn't think that someone could have this set.